terraform/modules/lambdas/common.py

```diff
@@ -0,0 +1,18 @@
+import json
```

Comment:
I've done a bit of commonisation here. Open to being told this is a terrible idea